ccl/changefeedccl: TestChangefeedContinuousTelemetryOnTermination failed

ccl/changefeedccl.TestChangefeedContinuousTelemetryOnTermination [failed](https://tanzanite.cluster.engflow.com/invocations/default/f516e900-d9ad-494d-b367-e793e7896d08?testReportRun=10&testReportShard=11&testReportAttempt=1#targets-Ly9wa2cvY2NsL2NoYW5nZWZlZWRjY2w6Y2hhbmdlZmVlZGNjbF90ZXN0) on master @ [a60fc9aa732dc25a36d14415cee29c41a59b48a7](https://github.com/cockroachdb/cockroach/commits/a60fc9aa732dc25a36d14415cee29c41a59b48a7):


```
=== RUN   TestChangefeedContinuousTelemetryOnTermination
    test_log_scope.go:170: test logs captured to: outputs.zip/logTestChangefeedContinuousTelemetryOnTermination3396966673
    test_log_scope.go:81: use -show-logs to present logs inline
    changefeed_test.go:5980: -- test log scope end --
test logs left over in: outputs.zip/logTestChangefeedContinuousTelemetryOnTermination3396966673
--- FAIL: TestChangefeedContinuousTelemetryOnTermination (48.22s)
=== RUN   TestChangefeedContinuousTelemetryOnTermination/pulsar
    helpers_test.go:864: making server as secondary tenant
    helpers_test.go:943: making pulsar feed factory
    helpers_test.go:1242: condition failed to evaluate within 45s: from helpers_test.go:1245: no logs found
    --- FAIL: TestChangefeedContinuousTelemetryOnTermination/pulsar (48.21s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>run=10</code>
 - <code>shard=11</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/cdc
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedContinuousTelemetryOnTermination.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>


Jira issue: CRDB-35963

ccl/changefeedccl.TestChangefeedContinuousTelemetryOnTermination [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/13901229?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/13901229?buildTab=artifacts#/) on master @ [7074888932021b3bec111c8cbf16e440e8070950](https://github.com/cockroachdb/cockroach/commits/7074888932021b3bec111c8cbf16e440e8070950):


```
=== RUN   TestChangefeedContinuousTelemetryOnTermination
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/905465f697dc04c39f7d8a82cb057bfe/logTestChangefeedContinuousTelemetryOnTermination3275126928
    test_log_scope.go:81: use -show-logs to present logs inline
    changefeed_test.go:5980: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/905465f697dc04c39f7d8a82cb057bfe/logTestChangefeedContinuousTelemetryOnTermination3275126928
--- FAIL: TestChangefeedContinuousTelemetryOnTermination (235.37s)
=== RUN   TestChangefeedContinuousTelemetryOnTermination/pulsar
    helpers_test.go:864: making server as secondary tenant
    helpers_test.go:943: making pulsar feed factory
    helpers_test.go:1242: condition failed to evaluate within 3m45s: from helpers_test.go:1245: no logs found
    --- FAIL: TestChangefeedContinuousTelemetryOnTermination/pulsar (235.36s)
```

Parameters:
 - <code>TAGS=bazel,gss</code>
 - <code>stress=true</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedContinuousTelemetryOnTermination.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

--- a/pkg/ccl/changefeedccl/helpers_test.go
+++ b/pkg/ccl/changefeedccl/helpers_test.go
@@ -881,7 +881,7 @@ func randomSinkTypeWithOptions(options feedTestOptions) string {
 		"pubsub":       1,
 		"sinkless":     2,
 		"cloudstorage": 0,
-		"pulsar":       1,
+		"pulsar":       0,
 	}
 	if options.externalIODir != "" {
 		sinkWeights["cloudstorage"] = 3

